web/src/app/components/LobbyPlayerList.tsx

```diff
@@ -6,12 +6,22 @@ interface LobbyPlayerListProps {
 
 export default function LobbyPlayerList({players} : LobbyPlayerListProps) {
     return (
-        <div className="inline-block overflow-auto rounded-lg bg-[#EAEAEA] px-4 py-2"> 
+        <div className="inline-block overflow-auto rounded-lg bg-[#EAEAEA] relative px-4 py-2"> 
             <h2 className="text-black">Connected Players ({Object.keys(players).length}/4)</h2>
             <ul>
-                {Object.entries(players).map(([id, {username}], index) => (
-                    <div key={id + username} className="mt-1">
-                         <li className="subtext text-black">{username}</li>  
+                {Object.entries(players).map(([id, {username, host}], index) => (
+                    <div key={id + username} className={`mt-1 ${host ? 'flex justify-items-center' : '' }`}>
+                        
+                        {host &&  <svg
```

Comment:
I think we probably will need a new icon for indicating the host, but this one can do for now I suppose.